panther_battery/src/adc_node.py

```diff
@@ -15,9 +13,12 @@
 
 
 class ADCNode:
+    BAT01_CHARGING_CURR_THRESH = 0.1
```

Comment:
```suggestion
    BAT_CHARGING_CURR_THRESH = 0.1
```
Since this parameter is used for bot batteries, I don't think there is the point specifying that `01` indicating its assignment to the battery 1